MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>title, axis labels</p><p> </p><p>what is the definition of the column "Urbanité Ruralié" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>